Chiazokam/Politico
# #163504428 create the delete party endpoint

#### What does this PR do?
This PR tests and creates the delete party Endpont

#### Description of Task to be completed?
- write tests for every test case
- implement admin check with middleware
- create the get one party endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-delete-party-163504428`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `DELETE` request to `http://localhost:3000/api/v1/parties/1`
- You should receive a successful response

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#163504428](https://www.pivotaltracker.com/story/show/163504428)

### Images
![image](https://user-images.githubusercontent.com/26940294/51914302-d54cd080-23d8-11e9-8450-2d0dccab243e.png)
